src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -24,8 +24,8 @@
 class SyliusApiExtension extends AbstractResourceExtension implements PrependExtensionInterface
 {
     protected $configFiles = array(
```

Comment:
Can we drop this too? We could use finder for loading all `*.xml` / `*.yml` files in `Resource/config` directory...